components/atom/progressBar/src/LineDoubleProgressBar/index.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomLineDoubleProgressBarV2'
+const CLASS_INDICATOR = `${BASE_CLASS}-indicator`
+const CLASS_CONTAINER_BAR = `${BASE_CLASS}-container`
+const CLASS_BAR = `${BASE_CLASS}-bar`
+const CLASS_BAR_ANIMATED = `${CLASS_BAR}--animated`
+const CLASS_EXTRA_BAR = `${BASE_CLASS}-extraBar`
+const CLASS_EXTRA_BAR_ANIMATED = `${CLASS_EXTRA_BAR}--animated`
+
+const Indicator = props => {
+  const {indicatorBottom, mainBarPercentage, indicatorTotal} = props // eslint-disable-line react/prop-types
```

Comment:
Why not this? `const Indicator = {indicatorBottom, mainBarPercentage, indicatorTotal}  => {`